strongbox-web-core/src/main/java/org/carlspring/strongbox/security/authentication/suppliers/ArtifactAuthenticationSupplier.java

```diff
@@ -0,0 +1,65 @@
+package org.carlspring.strongbox.security.authentication.suppliers;
+
+import org.carlspring.strongbox.configuration.ConfigurationManager;
+import org.carlspring.strongbox.storage.Storage;
+import org.carlspring.strongbox.storage.repository.Repository;
+
+import javax.annotation.Nonnull;
+import javax.inject.Inject;
+import javax.servlet.http.HttpServletRequest;
+
+import org.springframework.core.annotation.Order;
+import static org.carlspring.strongbox.web.Constants.ARTIFACT_ROOT_PATH;
+
+@Order(3)
+public abstract class ArtifactAuthenticationSupplier
```

Comment:
@LeifBrooks can you please rename it to `LayoutAuthenticationSupplier`?

Reply:
sure thing

Reply:
done